File: app/src/main/java/com/funnythingz/register/model/InAppPurchaseData.java
package com.funnythingz.register.model;

import com.google.gson.annotations.SerializedName;

/**
 * アプリ内課金注文情報
 */
public class InAppPurchaseData {
    // 注文番号
    @SerializedName("orderId")
    private String mOrderId;
    // アプリのパッケージ名
    @SerializedName("packageName")
    private String mPackageName;
    // 商品番号
    @SerializedName("productId")
    private String mProductId;
    // 購入日時
    @SerializedName("purchaseTime")
    private String mPurchaseTime;
    // 購入状態（0：購入 1：キャンセル 2：返金）
    @SerializedName("purchaseState")
    private String mPurchaseState;
    // 購入時に指定した任意の補足情報文字列
    @SerializedName("developerPayload")
    private String mDeveloperPayload;
    // 商品とユーザを結びつけるトークン
    @SerializedName("purchaseToken")
    private String mPurchaseToken;

    public InAppPurchaseData(String orderId, String packageName, String productId, String purchaseTime, String purchaseState, String developerPayload, String purchaseToken) {
        mOrderId = orderId;
        mPackageName = packageName;
        mProductId = productId;
        mPurchaseTime = purchaseTime;
        mPurchaseState = purchaseState;
        mDeveloperPayload = developerPayload;
        mPurchaseToken = purchaseToken;
    }

    public String getOrderId() {
        return mOrderId;
    }

    public String getPackageName() {
        return mPackageName;
    }

    public String getProductId() {
        return mProductId;
    }

    public String getPurchaseTime() {
        return mPurchaseTime;
    }

    public String getPurchaseState() {
        return mPurchaseState;
    }

    public String getDeveloperPayload() {
        return mDeveloperPayload;
    }

    public String getPurchaseToken() {
        return mPurchaseToken;
    }
}

File: app/src/main/java/com/funnythingz/register/model/InAppPurchaseData.kt
package com.funnythingz.register.model

/**
 * アプリ内課金注文情報
 */
class InAppPurchaseData(
        // 注文番号
        val orderId: String,
        // アプリのパッケージ名
        val packageName: String,
        // 商品番号
        val productId: String,
        // 購入日時
        val purchaseTime: String,
        // 購入状態（0：購入 1：キャンセル 2：返金）
        val purchaseState: String,
        // 購入時に指定した任意の補足情報文字列
        val developerPayload: String,
        // 商品とユーザを結びつけるトークン
        val purchaseToken: String)
